Alicerce/Sources/Logging/Log+ConsoleProvider.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  Log+ConsoleProvider.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ConsoleProvider: LogProvider {
+
+        public enum ConsoleOutput {
+            case print
+            case nslog
+        }
+
+        public var minLevel = Log.Level.error
+        public var formatter: LogItemFormatter = Log.ItemStringFormatter()
+        public var output = ConsoleOutput.print
```

Comment:
Completely agree with you. I think we should have this as a rule to NOT USER type inference in class properties/ attributes. I also think it's more readable, but honestly I thought it would be already a Swift best-practise.
We should compile some Swift Coding Style somewhere and refactor this accordingly.